src/clients/contracts/getters.ts

```diff
@@ -0,0 +1,88 @@
+import Web3 from 'web3';
```

Comment:
It would be nice to track this file as being moved in git

Reply:
I'm not too sure why GitHub doesn't track is as having been moved on the final result to be honest, since it did track it when I moved it: https://github.com/VenusProtocol/venus-protocol-interface/pull/296/commits/8c59e25b45909b02bb20cfb346b665dbeee9ca23#diff-43ab2a040e52b1a9646b6078715f89921951c1fbebd15d7ffa7b136d34f7b20a